cumulusci/core/keychain/base_project_keychain.py

```diff
@@ -147,45 +116,60 @@ def get_default_org(self):
                 return org, org_config
         return None, None
 
+    def get_org(self, name: str):
```

Comment:
Most of the changes to this file was me moving around the Public API methods to be at the top of their respective sections.